samples/snippets/src/test/java/com/google/cloud/examples/securitycenter/snippets/MuteFindingIT.java

```diff
@@ -0,0 +1,238 @@
+  private static Finding FINDING_1;
+  private static Finding FINDING_2;
+  private static Finding FINDING_3;
+
+  private static ByteArrayOutputStream stdOut;
+
+  // Check if the required environment variables are set.
+  public static void requireEnvVar(String envVarName) {
+    assertWithMessage(String.format("Missing environment variable '%s' ", envVarName))
+        .that(System.getenv(envVarName))
+        .isNotEmpty();
+  }
+
+  @BeforeClass
+  public static void setUp() throws IOException {
+    requireEnvVar("GOOGLE_APPLICATION_CREDENTIALS");
+    requireEnvVar("SCC_PROJECT_ID");
+    requireEnvVar("SCC_PROJECT_ORG_ID");
+
+    // Create mute rules.
+    CreateMuteRule.createMuteRule(String.format("projects/%s", PROJECT_ID), MUTE_RULE_CREATE);
+    CreateMuteRule.createMuteRule(String.format("projects/%s", PROJECT_ID), MUTE_RULE_UPDATE);
+    // Create Source.
+    SOURCE = createSource(ORGANIZATION_ID);
+    // Create Findings within the source.
+    String uuid = UUID.randomUUID().toString().split("-")[0];
+    FINDING_1 = createFinding(SOURCE.getName(), "1testingscc" + uuid);
+    FINDING_2 = createFinding(SOURCE.getName(), "2testingscc" + uuid);
+    FINDING_3 = createFinding(SOURCE.getName(), "3testingscc" + uuid);
+  }
+
+  @AfterClass
+  public static void cleanUp() {
+    final PrintStream out = System.out;
+    stdOut = new ByteArrayOutputStream();
+    System.setOut(new PrintStream(stdOut));
+    DeleteMuteRule.deleteMuteRule(PROJECT_ID, MUTE_RULE_CREATE);
+    assertThat(stdOut.toString()).contains("Mute rule deleted successfully: " + MUTE_RULE_CREATE);
+    DeleteMuteRule.deleteMuteRule(PROJECT_ID, MUTE_RULE_UPDATE);
+    assertThat(stdOut.toString()).contains("Mute rule deleted successfully: " + MUTE_RULE_UPDATE);
+    stdOut = null;
+    System.setOut(out);
+  }
+
+  public static Source createSource(String organizationId) throws IOException {
+    try (SecurityCenterClient client = SecurityCenterClient.create()) {
+
+      Source source =
+          Source.newBuilder()
+              .setDisplayName("Custom display name")
+              .setDescription("A source that does X")
+              .build();
+
+      CreateSourceRequest createSourceRequest =
+          CreateSourceRequest.newBuilder()
+              .setParent(String.format("organizations/%s", organizationId))
+              .setSource(source)
+              .build();
+
+      Source response = client.createSource(createSourceRequest);
+      System.out.println("Created source : " + response.getName());
+      return response;
+    }
+  }
+
+  public static Finding createFinding(String sourceName, String findingId) throws IOException {
+    try (SecurityCenterClient client = SecurityCenterClient.create()) {
+
+      Instant eventTime = Instant.now();
+
+      // The resource this finding applies to. The Cloud Security Command Center UI can link
+      // the findings for a resource to the corresponding Asset of a resource
```

Comment:
Re: Asset

Asset --> asset

Alternatively, link to the javadoc for the Asset class?